src/core/authorization/rules/authorization.rule.community.member.ts

```diff
@@ -8,12 +8,12 @@ import { IAuthorizationRule } from '@core/authorization/rules';
 import { AuthorizationCredential } from '@common/enums';
 
 export class AuthorizationRuleCommunityMember implements IAuthorizationRule {
-  communityID: number;
+  communityID: string;
   priority: number;
 
   constructor(parentArg: any, priority?: number) {
     this.communityID = parentArg.id;
-    if (this.communityID == -1) {
+    if (this.communityID === '-1') {
```

Comment:
everywhere else we have set / checked empty string, why is this different?

Reply:
https://github.com/cherrytwist/server/pull/998/commits/bd778338bf8e073641bec5c5cccf51962c8657b8